src/main/java/org/engine/process/performance/multi/EngingPerformanceMultiPeriods.java

```diff
@@ -68,14 +74,27 @@ protected void preExecute() throws Exception {
 	protected void postExecute() throws Exception {
 		
 		System.out.println("===postExecute");
+		
+		rowDataToSourceDiffTimeArray = new double[num_of_cycles*num_of_updates_per_cycle];
+		sourceToUpdateDiffTimeArray= new double[num_of_cycles*num_of_updates_per_cycle];
+		totalDiffTimeArray= new double[num_of_cycles*num_of_updates_per_cycle];
+		
+		rowDataToSourceDiffTimeCreateArray = new double[num_of_cycles];
+		sourceToUpdateDiffTimeCreateArray= new double[num_of_cycles];
+		totalDiffTimeCreateArray= new double[num_of_cycles];
+		
+		rowDataToSourceDiffTimeUpdateArray = new double[num_of_cycles*(num_of_updates_per_cycle-1)];
```

Comment:
I'm allergic to long variable names. Please use shorter names (about 3 words, unless you really need more).